billy-portugal/src/test/java/com/premiumminds/billy/portugal/test/services/builders/TestPTGenericInvoiceBuilder.java

```diff
@@ -98,7 +99,8 @@ public void doTest() {
                 .setSettlementDescription(mock.getSettlementDescription())
                 .setSettlementDiscount(mock.getSettlementDiscount()).setSourceId(mock.getSourceId())
                 .setTransactionId(mock.getTransactionId()).setSourceBilling(mock.getSourceBilling())
-                .setCustomerUID(mockCustomerEntity.getUID()).addPayment(builderPayment);
+                .setCustomerUID(mockCustomerEntity.getUID()).addPayment(builderPayment)
+                .setCreditOrDebit(GenericInvoice.CreditOrDebit.CREDIT);
```

Comment:
My expectation was to avoid this set outside of the manual builders. I see that this field is being set on validate, and that is being called on build.

Do you confirm?

Reply:
My mistake.